maestro-driver-ios/maestro-driver-iosUITests/Routes/XCTest/EventRecord.swift

```diff
@@ -20,9 +20,9 @@ struct EventRecord {
             .takeUnretainedValue() as! NSObject
     }
 
-    mutating func addPointerTouchEvent(at point: CGPoint, touchUpAfter: TimeInterval = defaultTapDuration) {
+    mutating func addPointerTouchEvent(at point: CGPoint, touchUpAfter: TimeInterval?) {
         var path = PointerEventPath.pathForTouch(at: point)
-        path.offset += touchUpAfter
+        path.offset +=  touchUpAfter ?? EventRecord.defaultTapDuration
```

Comment:
Nit: extra space

Reply:
Also, I'm not sure why you've moved the default argument to an optional. Default argument value is preferred, if the compiler allows it.

Reply:
Oh I see, the value you're passing is an optional. In that case, great!